run_gpu.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+. ./prereq_install.sh
+
+conda env create -f ./requirements_gpu.yml
```

Comment:
pls follow pattern from `run_cpu.sh` - use `source` instead of `.` and do an existing environment check

Reply:
fixed